Tests/Mapping/Driver/AbstractDriverTest.php

```diff
@@ -37,25 +38,23 @@ public function testFindMappingFileInSubnamespace()
         );
     }
 
-    /**
-     * @expectedException Doctrine\Persistence\Mapping\MappingException
-     */
     public function testFindMappingFileNamespacedFoundFileNotFound()
     {
         $driver = $this->getDriver([$this->getFixtureDir() => 'MyNamespace\MyBundle\Document']);
 
+        $this->expectException(MappingException::class);
```

Comment:
This line should be moved right before the one that triggers the exception (so probably the last one, right?)